lib/msf/core/auxiliary/report.rb

```diff
@@ -380,15 +380,16 @@ def report_web_vuln(opts={})
   # +filename+ and +info+ are only stored as metadata, and therefore both are
   # ignored if there is no database
   #
-  def store_loot(ltype, ctype, host, data, filename=nil, info=nil, service=nil)
+  # +changepath+ if true, store file according to host. e.g. ".msf4/loot/host/filename".
+  #
+  def store_loot(ltype, ctype, host, data, filename=nil, info=nil, service=nil, changepath=nil)
```

Comment:
```suggestion
  def store_loot(ltype, ctype, host, data, filename=nil, info=nil, service=nil, keeppath=false)
```
I think default value of the varaible is `false`, not `nil`, and `keeppath` is more accurately.